About me/projects.html

```diff
@@ -7,10 +7,16 @@
 	html {
 		background-color: #fff;
 	}
+	.pic {
+		margin: 15px 0px;
+	}
 	</style>
-</head>
+	</head>
 <body>
 <h1>Projects Links</h1>
 <a href="http://localhost:8080/projects/Karma/index_karma.html">Karma Clone Webpage</a>	
+<div class="pic">
+	<img src="KarmaSnip.jpg" alt="Karma" style="width:304px;height:228px;">
```

Comment:
As a rule of thumb, we want to avoid writing inline styles with the html '''style''' property. 

Remember that the rules for css specificity state that inline styles overwrite all other styles, which makes them difficult to maintain. We want to always leave room to override our styles should the need arise. 

Since this image is a portfolio item, why don't we give it a class that we can use to identify it as such. Let's give this image a class of '''.portfolio-item''' and then proceed to define the styles globally for this class.